### DISCUSSION TITLE
Architecture 란 무엇인가

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#20] [Q&A] [answered]
Architecture 란 무엇인가

**Comment by tjdud22** [comment, marked as answer]
아키텍처란
시스템의 구조, 동작 등을 정의하는 개념적인 모형으로 시스템의 목적을 달성하기 위해 시스템의 각 컴포넌트가 무엇이며 어떻게 상호작용 하는지, 정보가 어떻게 교환되는 지를 설명한다.
